gateway/conf/nginx.conf.liquid

```diff
@@ -43,7 +43,8 @@ http {
   tcp_nodelay        on;
   server_tokens off;
 
-
+  variables_hash_bucket_size 128;
+  variables_hash_max_size 1024;
```

Comment:
There is no point in declaring `variable_hash_max_size 1024;` because that is already the default value. We can see that in the log output attached in the JIRA.

I think we need to just follow Nginx documentation here:
>if nginx emits the message requesting to increase either hash max size or hash bucket size then the first parameter should first be increased.

```
variables_hash_max_size 2048;
```

If that fixes the issue we are good, if not then declare:

```
variables_hash_bucket_size 256;
```
At least this is how I understand what Nginx documentation is suggesting to do.

Reply:
My mistake with the PR, it got the THREESCALE-7942 together, just ignore the first commit.
I added `variables_hash_max_size` with the default value because both configurations are used together, keep it clear what's configured, and there's no problem to specify some variable with their default value.

Reply:
According to the doc:
```
The hash bucket size parameter is aligned to the size that is a multiple of the processor’s cache line size. This speeds up key search in a hash on modern processors by reducing the number of memory accesses
```

So, as I understand, the `variables_hash_bucket_size` is computed internally to be aligned with the  processor’s cache line size, it does not have a fixed default size. We should not set it.